index.php

```diff
@@ -1273,7 +1273,7 @@ public function logVersion() {
 }
 
 // Check for authentication
-$password = isset($_SERVER['HTTP_X_UPDATER_AUTH']) ? $_SERVER['HTTP_X_UPDATER_AUTH'] : '';
+$password = isset($_SERVER['HTTP_X_UPDATER_AUTH']) ? $_SERVER['HTTP_X_UPDATER_AUTH'] : (isset($_POST['updater-secret-input']) ? $_POST['updater-secret-input'] : '');
```

Comment:
I think `X-Updater-Auth` is only used here? https://github.com/search?q=org%3Anextcloud+X-Updater-Auth&type=Code Maybe its good to keep the `X-Updater-Auth` for backwards compatibility?

Reply:
I would keep it for now. I just added the option to also pass it via POST request additionally.